tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -447,3 +448,7 @@ def test_disruptive_during_pod_pvc_deletion_and_io(
         # Check ceph status
         ceph_health_check(namespace=config.ENV_DATA['cluster_namespace'])
         log.info("Ceph cluster health is OK")
+
+        log.info("Checking nodes status, bug #1745022")
+        wait_for_nodes_status(constants.NODE_READY)
```

Comment:
First parameter is node_names. status is second parameter.
```suggestion
        wait_for_nodes_status(status=constants.NODE_READY)
```

Missed this in first review.

Reply:
solved